tests/console/system_prepare.pm

```diff
@@ -87,7 +88,10 @@ sub run {
             diag "SUSEConnect --status-text locked: $out";
         }
         diag "SUSEConnect --status-text: $out";
-        assert_script_run "SUSEConnect --status-text | grep -v 'Not Registered'" unless get_var('MEDIA_UPGRADE');
+        if (!get_var('MEDIA_UPGRADE')) {
+            assert_script_run "SUSEConnect --status-text | grep -v 'Not Registered'";
```

Comment:
It is better to verify the system_prepare won't be run on system before SLE12 since SLES11SP4 don't support SUSEConnect.
And please run a verify log on opensuse, WDYT?

Reply:
This part  has been verified before. It has 'is_sle' condition, it has run 'SUSEConnect --status-text' before. So I don't think we need add opensuse to check.

Reply:
It won't run 11sp4 as it only run after migration